examples/minimal/packages/client-react/src/mud/snapSync/getSnapSyncRecords.ts

```diff
@@ -0,0 +1,48 @@
+import { RawTableRecord } from "@latticexyz/network";
+import { TableId } from "@latticexyz/utils";
+import snapSyncSystemAbi from "./snapSyncSystemAbi";
+import { Contract, Signer, providers } from "ethers";
+import { defineContractComponents } from "../contractComponents";
+
+export async function getSnapSyncRecords(
+  worldAddress: string,
+  contractComponents: ReturnType<typeof defineContractComponents>,
+  currentBlockNumber: number,
+  signerOrProvider: Signer | providers.JsonRpcProvider
+) {
+  const snapSyncContract = new Contract(worldAddress, snapSyncSystemAbi, signerOrProvider);
+
+  const chunkSize = 100;
+  const tableIds = Object.values(contractComponents).map((c) => c.metadata.contractId);
```

Comment:
I'd propose passing in the table ids (and grabbing them directly from the mud config in `setup.ts`)

Reply:
added this